src/App.js

```diff
@@ -91,21 +91,38 @@ function App() {
   }, [])
 
   useEffect(() => {
-    if (todoList.length > 0) {
+    if (todoList.length) {
       localStorage.setItem('savedTodoList', JSON.stringify(todoList))
     }
   }, [todoList])
 
-
   return (
-    <>
-      <h1>Todo List</h1>
-      <AddTodoForm onAddTodo={addTodo} />
-      {isloading ? <p>Loading...</p> :
-        <TodoList todoList={todoList} onRemoveTodo={removeTodo} />}
-    </>
-  );
+    <BrowserRouter>
+      <Routes>
+        <Route path="/" element={< Home />} />
+        <Route path="/new" element={< NewRoute />} />
+      </Routes>
+    </BrowserRouter>
+  )
+
+function Home() {
+    return (
+      <>
+        <h1>Todo List</h1>
+        <AddTodoForm onAddTodo={addTodo} />
+        {isloading ? <p>Loading...</p> :
+          <TodoList todoList={todoList} onRemoveTodo={removeTodo} />}
+      </>
+    )
+  }
+  
+  function NewRoute() {
+    return(
+      <>
+      <h1>New Todo List</h1>
+      </>
```

Comment:
Nitpick:
Since this return statement is only wrapping one element (`h1`), you can drop the fragments. If there are two or more elements that don't share a common parent element, then a fragment would be required.

Reply:
I did add the fragments in case later on I may need to add on to the route and didn't want to forget to put it.